elementary/clients/gcs/client.py

```diff
@@ -36,7 +38,8 @@ def send_report(self, local_html_file_path: str, remote_bucket_file_path: Option
                     new_name=bucket_path.join_path([bucket_report_folder_path, 'index.html']) if bucket_report_folder_path else 'index.html'
                 )
                 logger.info("Updated GCS bucket's website.")
-        except google.cloud.exceptions.GoogleCloudError:
+        except google.cloud.exceptions.GoogleCloudError as gcp_exception:
             logger.error('Failed to upload report to GCS.')
+            logger.error("Error trace: %s", gcp_exception)
```

Comment:
What do you think about adding the exception tracing message to the error message above?

Reply:
That sounds like a plan. Updated:
```python
            logger.error('Failed to upload report to GCS.\nError trace: %s')

```